frontend/src/components/sign-in-form/index.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import styles from './signin.module.scss';
+import { IUserLoginPayload } from 'healthcare-shared/common/interfaces';
+import { LoginPayloadKey } from 'healthcare-shared/common/enums';
+import { loginSchema } from 'healthcare-shared/helpers';
+import { Link } from 'components/common';
+import { AppRoute } from 'common/enums';
+
+const SignInPage: React.FC = () => {
+  const { register, handleSubmit, errors } = useForm<IUserLoginPayload>({
+    resolver: yupResolver(loginSchema),
+  });
+
+  const onSubmit = (formValues: IUserLoginPayload) => {
```

Comment:
```suggestion
  const handleFormSubmit = (formValues: IUserLoginPayload) => {
```

Local names for cb - `handle<Element><Event>`
Example - handleFormSubmit

Props names for cb - `on<Element><Event>`
Example - `onFormSubmit`